### ISSUE TITLE
[Programmatic Access – Make Code - Import] Incorrect role is defined as 'menu item' for buttons like: 'Create new' present on 'pop up screen'.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3754]
User Experience:
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is a link/menu items or a button or whether it is a non-interactive control. Users will understand it to be a interactive control as screen reader announces link while navigating in scan mode and hence it will not be a good usable experience for the users.

Note: User credentials should NOT be included in the bug.

Pre-Requisite:
Open F12 Developer tool.

Repro Steps:

Launch application URL
Home page will open.
Navigate to "Import" button and select it.
Navigate through the controls appearing under "Import" dialog.
Navigate to "Your GitHub Repo" option and  Select it.
Navigate and select "Sign In" control under Sign in with GitHub and navigate to all the controls of dialogue "Clone or create your own GitHub repo
Verify the issue.

Actual Result:
Incorrect role is defined as 'menu item' for buttons like: 'Create new' present on 'pop up screen' screen.

Expected Result:
Correct role should be define as 'Button' instead of 'menu item' for buttons like: 'Create new' present on 'pop up screen' screen.

Note:

1. Same issue exists on Firefox + NVDA & Chrome + JAWS.
2. Same issue exists throughout the application under the pop up.
3. Same issue exists for  'Exit tutorial' under' screen

WCAG Reference:
[WCAG 4.1.2 – Name, Role, Value
](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html%7D)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG4.1.2; #Win10-Edge(Chromium)

![Uploading WCAg4.1.2_Incorrect role is defined as 'menu item' for buttons.png…]()
